utility/src/main/java/org/oppia/util/firebase/FirebaseModule.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+class FirebaseModule {
```

Comment:
Let's not call this FirebaseModule, maybe LogReportingModule?  (I am generally bad with names so @rt4914 @BenHenning do you have any suggestions )

Reply:
Even I am not good with naming but `LogReportingModule` sounds better than `FirebaseModule`.

Also @Sarthak2601 Please add KDoc for every new class or public functions.

Reply:
Done